3회차/문재윤/1_소득불균형.py

```diff
@@ -1,3 +1,15 @@
-import sys
+# import sys
 
-sys.stdin = open("_소득불균형.txt")
+# sys.stdin = open("_소득불균형.txt")
+n = int(input())
+g = 0
+for j in range(n):
+    g += 1
+    f = int(input())
+    money = list(map(int,input().split()))
+    average_ = sum(money)/len(money)
+    co = 0
+    for i in money:
+        if i <= average_:
+            co += 1
+    print(f'#{g} {co}')
```

Comment:
새로운 변수 `g`를 사용하는 대신, `for`문의 `j`를 이용해 출력하는 방법도 있습니다!
```suggestion
    print(f'#{j + 1} {co}')
```